web/pages/review/cards.tsx

```diff
@@ -0,0 +1,107 @@
+import { useAuth } from "react-oidc-context";
+import { useQuery } from "@tanstack/react-query";
+import {
+    reviewFidsKey,
+    reviewModesKey,
+    useSessionStorage,
+} from "@src/useStorage";
+import { getRequest } from "@src/apiRequest";
+import ProtectedRoute from "@components/ProtectedRoute";
+import Flip from "@components/routeReview/Flip";
+import { NextFlashcardGetReq } from "@src/types/NextFlashcardGetReq";
+import { NextFlashcardGetResp } from "@src/types/NextFlashcardGetResp";
+import { FlashcardFromIDResp } from "@src/types/FlashcardFromIDResp";
+import { FlashcardFromIDGet } from "@src/types/FlashcardFromIDGet";
+import Button from "@components/Button";
+/*
+This page is accessed when the user has set modes and flashcards in localStorage. 
+This page sends a request to /review/next to get the next 'packet' of
+upcomingReview info. Then, it requests the flashcard info from ID via 
+/flashcard/fromid and displays this.
```

Comment:
For v2, might be worth doing a bulk request endpoint, so we can get the details of the next packet all at once

Reply:
Yeah, definitely.